packages/i18n/README.md

```diff
@@ -129,41 +101,91 @@ Just import the `t` function and use it to translate the key.
 ```typescript
 import { t } from '@extension/i18n';
 
-console.log(t('helloWorld')); // Hello, World!
+console.log(t('loading')); // Loading...
 ```
 
 ```typescript jsx
 import { t } from '@extension/i18n';
 
 const Component = () => {
   return (
-    <div>
-      {t('helloWorld')} // Hello, World!
-    </div>
+    <button>
+      {t('toggleTheme')} // Toggle Theme
+    </button>
   );
 };
 ```
 
 ### Placeholders
 
-If you want to use placeholders, you can pass the second argument as a string or an array.
+If you want to use placeholders, you can use the following format.
+
+> For more information, see the [Message Placeholders](https://developer.chrome.com/docs/extensions/how-to/ui/localization-message-formats#placeholders) section.
+
+`locales/en/messages.json`
+
+
```

Comment:
1 additional blank space, relative to example below 😄

Reply:
- https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/613/commits/2762ac8f4238c5f2508a65500bde0f17dee18b52

😄